### ISSUE TITLE
RocksJava stats.getTickerCount() returns zero

### ISSUE THREAD

**Issue by mavigdor** [facebook/rocksdb#1937]
Hi,
I'm trying to get statistics using RocksJava but `stats.getTickerCount()` returns zero.
Anyone aware of this issue?

This is my code:
```java
options = new Options().setCreateIfMissing(true);
options.createStatistics();
options.setStatsDumpPeriodSec(1);
stats = options.statisticsPtr();
db = RocksDB.open(options, db_path);
assert (db != null);
// create column family
columnFamilyHandle = db.createColumnFamily(
new ColumnFamilyDescriptor("new_cf".getBytes(),
new ColumnFamilyOptions()));
assert (columnFamilyHandle != null);
for(int i = 0; i < 1000000; i++) {
db.put(columnFamilyHandle, new WriteOptions(),
"key".getBytes(), "value".getBytes());
System.out.println("stats = " + stats.getTickerCount(TickerType.BYTES_WRITTEN));
}
```

**Comment by adamretter** [comment]
So my first question would be, if you write the equivalent C++ code, do you also get 0?

**Comment by yaelna** [comment]
Hi Adam,
I saw that you have a commit in your forked repo that is linked with Issue #2374,
I've built the jar from your repo from branch improve-stats-enums and it has solved our problem :)
Are you planning on submitting a pull request soon? we are looking forward for a rocksdb release with this fix.
thank you, 
Yael

**Comment by gfosco** [comment]
Closing this via automation due to lack of activity.  If you'd like this to be re-opened, please just comment here and we'll open it back up.

**Comment by adamretter** [comment]
@yaelna this was previously merged around 000bf0a
